app/pages/ukrentaroom/adjustments/RaRAdjustmentsCompletePage.scala

```diff
@@ -16,12 +16,12 @@
 
 package pages.ukrentaroom.adjustments
 
-import pages.QuestionPage
+import pages.{PageConstants, QuestionPage}
 import play.api.libs.json.JsPath
 
 case object RaRAdjustmentsCompletePage extends QuestionPage[Boolean] {
 
-  override def path: JsPath = JsPath \ toString
+  override def path: JsPath = JsPath \ PageConstants.rentARoomSectionFinished \ toString
```

Comment:
We can include this in the RaRAdjustments (PageConstants.RaRAdjustments) section